batch/batch/worker.py

```diff
@@ -146,7 +174,20 @@ def container_config(self):
                            'Memory': self.memory_in_bytes}
         }
 
-        env = self.spec.get('env')
+        env = self.spec.get('env', [])
+
+        if self.port is not None:
+            assert self.host_port is not None
+            config['PortBindings'] = {
+                f'{self.port}/tcp': [{
+                    'HostIp': '',
```

Comment:
I can't find documentation on the docker API. Can you point me to the docs that explain what empty string means for HostIp?

Reply:
Of course docker isn't documented to this level of detail.  In general, the API docs are here (create container, for example): https://docs.docker.com/engine/api/v1.40/#operation/ContainerCreate

Normally, I figure out the config I want by using the docker command line to create an example container and then inspect the configuration it created with `docker inspect`.  That's where this came from.  This also seems sensible, and I believe empty string means bind all host interfaces.